modules/msf_integration.py

```
@@ -0,0 +1,410 @@
+            raise RuntimeError("MSF载荷生成超时")
+        except Exception as e:
+            raise RuntimeError(f"MSF载荷生成错误: {str(e)}")
+    
+    def create_listener_command(self, payload_type: str, ip: str, port: int) -> str:
+        """生成对应的监听器命令"""
+        if 'meterpreter' in payload_type:
+            # Meterpreter监听器
+            if 'http' in payload_type:
+                transport = 'reverse_http' if 'https' not in payload_type else 'reverse_https'
+            else:
+                transport = 'reverse_tcp'
```

Ruff (0.12.2)

Local variable `transport` is assigned to but never used

Remove assignment to unused variable `transport`

(F841)